packages/expo-video/ios/VideoModule.swift

```diff
@@ -3,6 +3,9 @@
 import ExpoModulesCore
 
 public final class VideoModule: Module {
+  internal static let nowPlayingManager = NowPlayingManager()
+  internal static let videoViewManager = VideoViewManager()
```

Comment:
Hmm, we probably don't need to have them defined here. Instead these classes could be static or could be singleton-like (i.e. provide a single instance through `NowPlayingManager.shared`, it's quite common practice in Swift and Objective-C).